tests/tests_fabric/run_standalone_tasks.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# THIS FILE ASSUMES IT IS RUN INSIDE THE tests/tests_fabric DIRECTORY
+export PYTHONPATH="${PYTHONPATH}:$(pwd)"
+export PYTHONPATH="${PYTHONPATH}:$(pwd)/.."
+
+MAX_RETRIES=3
+
+retry_command() {
+  local command="$@"
+  local exit_code=1
+  for ((i=1; i<=$MAX_RETRIES; i++))
+  do
+    echo "Run attempt: $i"
+    eval $command
+    exit_code=$?
+    if [ $exit_code -eq 0 ]; then
+      echo "Successfully ran: $command"
+      break
+    fi
+    echo "Attempt $i failed."
+  done
+  if [ $exit_code -ne 0 ]; then
+    echo "Failed after $MAX_RETRIES attempts: $command"
```

Comment:
I tested the failure case in an early commit by inserting an `assert False` into the parity test. You can see the CI fails properly with exit code 1